project_restorer/src/main/scala/OwnMessageProcessor.scala

```diff
@@ -0,0 +1,49 @@
+import com.gu.multimedia.storagetier.framework.MessageProcessorConverters._
```

Comment:
Think that this file should be called `PlutoCoreMessageProcessor` to keep the naming convention working :)
`OwnMessageProcessor` implies that the messages it's handling are coming from itself

Reply:
Fixed :)